Tasks/ServiceFabricDeploy/task.json

```diff
@@ -413,7 +413,7 @@
         "SFSDK_AppTypeMismatch": "Application type of application '{0}' doesn't match the application type in the application manifest of the new application package. Please ensure that the application being upgraded has the same application type.",
         "SFSDK_UpgradeInProgressError": "An upgrade for the application '{0}' is already in progress.",
         "SFSDK_StartAppUpgrade": "Start upgrading application...",
-        "SFSDK_UnregisterAppTypeOnUpgradeFailure": "Unregistering application type '{0}' and version '{1}'...",
+        "SFSDK_UnregisterAppTypeOnUpgradeFailure": "Unregistering process of application type '{0}' and version '{1}' got failed",
```

Comment:
You also need to update the version in the manifest, build and commit the generated task.loc.json

Reply:
Sure I'll do that when I merge this PR with few others together.